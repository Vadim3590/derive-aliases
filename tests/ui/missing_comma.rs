pub mod derive_alias {
    derive_aliases::define! {
        Eq = ::core::cmp::Eq, ::core::cmp::PartialEq;
        AndMore = ::std::hash::Hash  ..Ord, ::core::clone::Clone, ::core::marker::Copy, ::core::default::Default;
        //                         ^ missing `,`
    }
}

fn main() {}
