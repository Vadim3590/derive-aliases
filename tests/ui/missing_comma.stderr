error: expected `;` or `,`
 --> tests/ui/missing_comma.rs:4:38
  |
4 |         AndMore = ::std::hash::Hash  ..Ord, ::core::clone::Clone, ::core::marker::Copy, ::core::default::Default;
  |                                      ^
